### ISSUE TITLE
What if the body of the `/users` request is malformed?

### ISSUE THREAD

**Issue by jadudm** [jadudm/a-homework-assignment#10]
What if I get a malformed response from the `/auth` endpoint?

That is, I could successfully get a token, successfully make a request against `/users`, and then the response could be malformed.

This may be too broad a ticket. But:

* What if the response body is `""`? (The empty string?)
* What if it is a single character or string of characters?
* What if I get something where it is a list of integers (one per line), but one of them is malformed? Specifically, what if one is a float?
* What if I get something that looks like a list of integers, but one of them is a string with quotes?
* What if I get something that looks like a list of integers, but one of them has a bit flipped, so it comes through as an `x`? (This is not a single bit flip, but whatevs.)

In short, before I do a `json.dumps()`, I have to make sure the response body can be dumped.
